### DISCUSSION TITLE
Guidance on importing a .dll

### DISCUSSION THREAD

**Discussion by angelos3lex** [microsoft/react-native-windows#12805] [Q&A] [answered]
Hey, I have a dll which I want to import in my react-native-windows project.
I have setup the bridge according to the FancyMath example as shown in the docs, and I'm successfully able to call the add method in .cpp file.

Now I want to import a dll. According to the ddl docs, they import it as:
#import "C:\\<path>\\dllName.dll"
But this is provoking errors on run about `IUnknown': ambiguous symbol` 

I tried to use `LoadLibrary` as `HINSTANCE hDLL = LoadLibrary(TEXT("C:\\<path>\\dllName.dll"));` 
But again I get another error:  `C3861: 'LoadLibrary': identifier not found`, even if I correctly have `#include <Windows.h>` in my headers.

At this point I'm not sure how should I proceed as I don't see any relevant documentation.

Also, is it possible to load it from the system path or should I copy the dll inside the windows project?

**Comment by chrisglein** [comment, marked as answer]
Less of a React Native for Windows question and more of a native Windows development question, but maybe these links can help?
- https://microsoft.github.io/react-native-windows/docs/native-code
- https://learn.microsoft.com/en-us/visualstudio/ide/managing-references-in-a-project?view=vs-2022
- https://stackoverflow.com/questions/59404143/how-to-add-dll-file-into-a-uwp-project
- https://stackoverflow.com/questions/38947820/including-a-third-party-dll-in-a-uwp-application-step-by-step
